apps/studio/components/interfaces/Support/SupportForm.tsx

```diff
@@ -428,19 +435,44 @@ const SupportForm = ({ setSentCategory }: SupportFormProps) => {
                 </div>
               )}
 
-            {subscription?.plan.id === 'free' && values.category !== 'Login_issues' && (
+            {subscription?.plan.id !== 'enterprise' && values.category !== 'Login_issues' && (
               <div className="px-6">
                 <InformationBox
                   icon={<IconAlertCircle strokeWidth={2} />}
+                  defaultVisibility={true}
+                  hideCollapse={true}
                   title="Expected response times are based on your project's plan"
                   description={
                     <div className="space-y-4 mb-1">
-                      <p>
-                        Free plan support is available within the community and officially by the
-                        team on a best efforts basis, though we cannot guarantee a response time.
-                        For a guaranteed response time we recommend upgrading to the Pro plan.
-                        Enhanced SLAs for support are available on our Enterprise Plan.
-                      </p>
+                      {subscription?.plan.id === 'free' && (
+                        <p>
+                          Free plan support is available within the community and officially by the
+                          team on a best efforts basis. For a guaranteed response we recommend
+                          upgrading to the Pro plan. Enhanced SLAs for support are available on our
+                          Enterprise Plan.
+                        </p>
+                      )}
+
+                      {subscription?.plan.id === 'pro' && (
+                        <p>
+                          Pro Plan includes email-based support. You can expect an answer within 1
+                          business day in most situation for all severities. We recommend upgrading
+                          to the Team plan for prioritized ticketing for all issues and prioritized
```

Comment:
```suggestion
                          to the Team plan for prioritized ticketing on all issues and prioritized
```

nit - to not repeat 'for'